### PR TITLE
[Enhancement]- Fix log pollution

### PR DESCRIPTION
**Pull request by drivebyer**
**Description**

Fixes https://github.com/OT-CONTAINER-KIT/redis-operator/issues/430

Add log verbosity param && level down some logger.

**Type of change**

* Bug fix (non-breaking change which fixes an issue)
* New feature (non-breaking change which adds functionality)
* Breaking change (fix or feature that would cause existing functionality to not work as expected)

**Checklist**

- [x] Testing has been performed
- [x] No functionality is broken
- [x] Documentation updated

### CODE REVIEW

**Comment by shubham-cmyk** [comment]
@drivebyer 
If we change the level for the logger from here https://github.com/OT-CONTAINER-KIT/redis-operator/blob/master/k8sutils/redis.go#L431

It would be much simpler to handle

**Comment by drivebyer** [comment]
> If we change the level for the logger from here https://github.com/OT-CONTAINER-KIT/redis-operator/blob/master/k8sutils/redis.go#L431

yes, I have already change the logger level from ZERO to ONE, along with many other loggers.

File: main.go
Comment (shubham-cmyk) [inline main.go:1]: In this file you should also add a way to control the verbosity via env var `LOG_LEVEL` could be name and args could be `--log-level`

**Review by shubham-cmyk — COMMENTED** [review]
Also @drivebyer Changing log level won't just solve the problem you have to move many of the log that we were marking on the info to the debug

File: main.go
Comment (drivebyer) [reply main.go:1]: Actually, i find the `zap-log-level` already added by `opts.BindFlags(flag.CommandLine)`.

Comment (shubham-cmyk) [reply main.go:1]: Okay why you removed that ? 
You should add that

**Review by shubham-cmyk — APPROVED** [review]
@drivebyer Great Work! Thanks a lot for PR

**Pull request merged**